content/hi/docs/reference/glossary/image.md

```diff
@@ -0,0 +1,17 @@
+---
+title: इमेज(Image)
+id: image
+date: 2018-04-12
+full_link:
+short_description: >
+  यह एक कंटेनर का एक संग्रहित उदाहरण है और इसमें एप्लिकेशन चलाने के लिए आवश्यक सॉफ़्टवेयर का एक समूह होता है।
+aka:
+tags:
+  - fundamental
+---
+
+एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} का संग्रहित उदाहरण जिसमें एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का एक समूह होता है।
+
+<!--more-->
+
+सॉफ़्टवेयर पैकेज करने का एक तरीका जिसे एक कंटेनर रजिस्ट्री में संग्रहीत किया जा सकता है, स्थानीय सिस्टम पर खींचा(pull) जा सकता है और एक एप्लिकेशन के रूप में चलाया जा सकता है। इमेज में निहित मेटाडेटा यह दर्शाता है कि कौन सा निष्पादन योग्य चल रहा है, इसे किसने बनाया है, और अन्य जानकारी देता है।
```

Comment:
Typo. Otherwise LGTM
```suggestion
सॉफ़्टवेयर पैकेज करने का एक तरीका जिसे एक कंटेनर रजिस्ट्री में संग्रहित किया जा सकता है, स्थानीय सिस्टम पर खींचा(pull) जा सकता है और एक एप्लिकेशन के रूप में चलाया जा सकता है। इमेज में निहित मेटाडेटा यह दर्शाता है कि कौन सा निष्पादन योग्य चल रहा है, इसे किसने बनाया है, और अन्य जानकारी देता है।
```